swagger_server/itm/itm_ta1_controller.py

```diff
@@ -5,38 +5,40 @@
 from swagger_server.models.probe_response import ProbeResponse  # noqa: F401,E501
 from swagger_server.models.alignment_results import AlignmentResults  # noqa: F401,E501
 from swagger_server.models.alignment_target import AlignmentTarget  # noqa: F401,E501
+from swagger_server import config_util
 
-class ITMTa1Controller:
-    ADEPT_PORT = os.getenv("ADEPT_PORT")
-    if (ADEPT_PORT is None or ADEPT_PORT == ""):
-        ADEPT_PORT = '8081'
-    SOARTECH_PORT = os.getenv("SOARTECH_PORT")
-    if (SOARTECH_PORT is None or SOARTECH_PORT == ""):
-        SOARTECH_PORT = '8084'
 
-    def __init__(self, alignment_target_id, scene_type, alignment_target = None):
+class ITMTa1Controller:
+    
+    config_util.check_ini()
+    config = config_util.read_ini()[0]
+    
+    
+    ADEPT_URL = config['DEFAULT']['ADEPT_URL']
+    SOARTECH_URL = config['DEFAULT']['SOARTECH_URL']
+    
+    def __init__(self, alignment_target_id, scene_type, config, alignment_target = None):
         self.session_id = ''
         self.alignment_target_id = alignment_target_id
         self.alignment_target = alignment_target
-        self.host, self.port = ITMTa1Controller.get_contact_info(scene_type=scene_type)
+        self.host_port = ITMTa1Controller.get_contact_info(scene_type=scene_type)
 
     @staticmethod
     def get_contact_info(scene_type):
-        port = ITMTa1Controller.ADEPT_PORT if scene_type == 'adept' else ITMTa1Controller.SOARTECH_PORT
-        host =  os.getenv("ADEPT_HOSTNAME") if scene_type == 'adept' else os.getenv("SOARTECH_HOSTNAME")
-        if host is None or host == "":
-            host = "localhost"
-        return host, port
-
+        host_port = ITMTa1Controller.ADEPT_URL if scene_type == 'adept' else ITMTa1Controller.SOARTECH_URL
+        # Technically this should never be hit since configs are mandatory but just in case
+        if host_port is None or host_port == "":
+            host_port = "localhost"
+        return host_port
     @staticmethod
     def get_alignment_data(scene_type):
-        (host, port) = ITMTa1Controller.get_contact_info(scene_type=scene_type)
+        host_port = ITMTa1Controller.get_contact_info(scene_type=scene_type)
         target_id_path = 'alignment_target_ids' if scene_type == 'adept' else 'alignment_targets'
-        url = f"http://{host}:{port}/api/v1/{target_id_path}"
+        url = f"http://{host_port}/api/v1/{target_id_path}"
```

Comment:
The `http://` needs to be removed as the URL contains these.  That was the problem with the old way-- it "forced" http.
e.g., `url = f"{url}/api/v1/{target_id_path}"` and so on.